index.js

```diff
@@ -1,10 +1,12 @@
-var semver = require('semver')
+import fs from 'node:fs';
 
-function getNextTarget (runtime, targets) {
+import semver from 'semver';
+
+export function _getNextTarget (runtime, targets) {
```

Comment:
This is a nit, but I wonder if we should remove `_getNextTarget` from `index.js` by moving it to its own util file and export it from there and _not_ export it from `index.js` so that it isn't part of the ultimate public API.

By the rules, it'd be a breaking change so I think it might fit in the scope of this PR. Otherwise, we could add it as a sneaky `fix:` later on.

Reply:
Good suggestion! Addressed in `72d0bf0`, and that also helped me catch a SNAFU where `index.js` was no longer being included after removing `main` in `package.json`. I've updated the upgrade checklist to point out this potential footgun. 👍